src/BoundaryConditions/fill_halo_regions_open.jl

```diff
@@ -1,39 +1,50 @@
+struct OrlanskiBoundary{G}
```

Comment:
Probably `OrlanskiCondition` is the right term here, since this is intended to be used in the `condition` property.

Btw another possibility is to simply use "Orlanski" as a synonym for "Open", and add properties to the `Open` classification. Not sure what's best.

Reply:
I don't think "Open" always means "Orlanski" though since Orlanski is just one way of approximating the boundary point but e.g. specifying the value (as I have done for in flows) is another valid "Open" boundary?

Reply:
I see what you're saying. In that case, perhaps just using either `Orlanski` or `OrlanskiCondition` is the improvement we need here.

You can also introduce `AbstractOpenClassification` of which `Orlanski` and `Open` are subtypes.